pkg/yang/identity_test.go

```diff
@@ -128,14 +128,14 @@ func TestIdentityExtract(t *testing.T) {
 			}
 
 			if ti.baseName != "" {
-				if ti.baseName != thisID.Base.Name {
+				if ti.baseName != thisID.Base[0].Name {
```

Comment:
Thanks for the contribution! Could you add a test case here for multiple base statements?

Reply:
I've added a test case for multiple bases.

In order to minimize changes to the existing test cases I added a new property `baseNames: []string` instead of changing the type of the existing `baseName` property in the test utility struct. If required, I can change that, of course.

Reply:
Thanks for adding the test case. Could you remove `baseName` please? It looks like just using `baseNames []string` can improve the code style.

I'm thinking of moving the `[]string` generation to a helper to make this easier, and then using the pattern here:
https://github.com/hslatman/goyang/blob/multiple-base-identities-1.1-v2/pkg/yang/node_test.go#L60
We're trying to use the new `cmp` package in tests whenever possible over `reflect.DeepEquals`.

Reply:
I've removed the `baseName` property in favor of `baseNames` and implemented the `cmp` package, resulting in a different kind of error message when the test fails, but I think it's more useful showing the actual diff. 

The utility function is usable in tests now. I don't know if this should be exposed in some way; as a method on `*Identity`, for example, or to just keep it like this.

Reply:
Nope, just keeping it here is fine, `cmp` is intended to only be used in tests.